working/manual_changes/2021/2021-03-29.sql

```diff
@@ -0,0 +1,29 @@
+DO $_$
+BEGIN
+	PERFORM vocabulary_pack.AddNewRelationship(
+	pRelationship_name			=>'Concepts with reference to variant',
+	pRelationship_id			=>'Has variant',
+	pIs_hierarchical			=>0,
+	pDefines_ancestry			=>0,
+	pReverse_relationship_id	=>'Variant of',
+	pRelationship_name_rev		=>'Variant refer to concept',
+	pIs_hierarchical_rev		=>0,
+	pDefines_ancestry_rev		=>0
+);
+END $_$;
+
+
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.AddNewVocabulary(
+	pVocabulary_id			=> 'OncoKB',
+	pVocabulary_name		=> 'Oncology Knowledge Base (MSK)',
+	pVocabulary_reference	=> 'https://www.oncokb.org/',
+	pVocabulary_version		=> NULL,
```

Comment:
Do we have at least a date of the sources?

Reply:
Date is requested from source providers, I'm sure it can be released without it until we don't get an answer on it.